packages/wonder-blocks-tooltip/components/tooltip.integration.test.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+import * as React from "react";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+import Tooltip from "./tooltip.js";
+import TooltipBubble from "./tooltip-bubble.js";
+import TooltipAnchor from "./tooltip-anchor.js";
+import {TooltipDisappearanceDelay} from "../util/constants.js";
+
+describe("tooltip integration tests", () => {
+    beforeEach(() => {
+        unmountAll();
+        jest.useFakeTimers();
+        // jest.clearAllMocks().resetModules();
+    });
+
+    it("should set timeoutId be null when TooltipBubble is active", () => {
+        // Arrange
+        const wrapper = mount(
+            <Tooltip content="hello, world">an anchor</Tooltip>,
+        );
+        const anchor = wrapper.find(TooltipAnchor).getDOMNode();
+
+        // Act
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseenter"));
+        // There's a 100ms delay before TooltipAnchor calls _setActiveState with
+        // instant set to true.  This second call is what actually triggers the
+        // call to this.props.onActiveChanged() which updates Tooltip's active
+        // state.
+        jest.runAllTimers();
+        // Since the call to update Tooltip's active state happens in a timeout
+        // we need to call update b/c it happens outside of the normal React lifecycle
+        // methods.
+        wrapper.update();
+
+        // Assert
+        expect(wrapper).toContainMatchingElement(TooltipBubble);
+        expect(wrapper).toHaveState("timeoutID", null);
+    });
+
+    it("should set a timeout on mouseleave on TooltipAnchor", () => {
```

Comment:
I re-enabled this test and update it to use similar techniques as the first test.